docs/transaction.rst

```diff
@@ -0,0 +1,57 @@
+Transaction Operations
+======================
+
+Transact operations are supported using context managers. The DynamoDB API has limits on the number of items in
```

Comment:
```suggestion
Transaction operations are supported using context managers. The DynamoDB API has limits on the number of items in
```

Reply:
I'd probably start with describing what transaction operations are, and only later go into the "we don't paginate yet" caveat.